interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+					</TD>
+	</TR>
+
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('CVS:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="CVS1" name="CVS1" type="checkbox" value="on" >
+			<label for="CVS1"> <?php echo xlt('Chest Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="CVS2" name="CVS2" type="checkbox" value="on" >
+			<label for="CVS2"> <?php echo xlt('Palpitations');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="CVS3" name="CVS3" type="checkbox" value="on" >
+			<label for="CVS3"> <?php echo xlt('Leg Pain w/Exercise');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="CVS_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>		
+		
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('GI:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="GI1" name="GI1" type="checkbox" value="on" >
+			<label for="GI1"> <?php echo xlt('Appetite Decrease');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI2" name="GI2" type="checkbox" value="on" >
+			<label for="GI2"> <?php echo xlt('Anorexia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI3" name="GI3" type="checkbox" value="on" >
+			<label for="GI3"> <?php echo xlt('Dysphagia');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI4" name="GI4" type="checkbox" value="on" >
+			<label for="GI4"> <?php echo xlt('Heartburn');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI5" name="GI5" type="checkbox" value="on" >
+			<label for="GI5"> <?php echo xlt('Nausea');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI6" name="GI6" type="checkbox" value="on" >
+			<label for="GI6"> <?php echo xlt('Vomiting');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI7" name="GI7" type="checkbox" value="on" >
+			<label for="GI7"> <?php echo xlt('Hematemesis');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI8" name="GI8" type="checkbox" value="on" >
+			<label for="GI8"> <?php echo xlt('Abdominal Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI9" name="GI9" type="checkbox" value="on" >
+			<label for="GI9"> <?php echo xlt('Jaundice');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI10" name="GI10" type="checkbox" value="on" >
+			<label for="GI10"> <?php echo xlt('Diarrhea');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GI11" name="GI11" type="checkbox" value="on" >
+			<label for="GI11"> <?php echo xlt('Constipation');?></label>
+		</span>
+		</TD>	
+	
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation